### ISSUE TITLE
Add heights and weight in participants.tsv

### ISSUE THREAD

**Issue by jcohenadad** [spine-generic/data-multi-subject#56]
The rationale for adding this information is to investigate potential correlates with cord CSA.

A suggestion would be to insert the following columns after "age" (example of value types are shown in the 2nd row, note: columns after date_of_scan are not shown for clarity):

| participant_id | sex | age | height | weight | date_of_scan |
|----------------|-----|-----|--------|--------|--------------|
| sub-amu01      | M   | 28  | 180    | 70     | 2019-02-12   |

Metric system is assumed, with height in cm and weight in kg.

Suggestions/feedback welcome

**Comment by renelabounek** [comment]
I would suggest to add there also columns **height_record** and **weight_record** with 3 possible choices:

choice 1 - **questionnaire** (i.e. approximate values with potential low accuracy)
choice 2 - **measured_prior_MRI** (i.e. exactly measured values)
choice 3 - **no_record** (i.e. not recorded or not shared information by the contributor)

I would also suggest to add units into the table header.

participant_id | sex | age [years] | height [cm] | weight [kg] | date_of_scan | height_record | weight_record
-- | -- | -- | -- | -- | -- | -- | --
sub-amu01 | M | 28 | 180 | 70 | 2019-02-12 | questionnaire | questionnaire

Initially, we have not planned to acquire this data, so current results will be biased by the questionnaire accuracy. In future database contributions, pepole can start to acquire precise values prior the MRI and then we will need to have an option to filter out the precise or approximate values.

Here is the preliminary observation (at cmrra+cmrrb Siemens Prisma-fit 13-sample dataset) reasoning this decision (r - Pearson correlation coefficient, p - p-value of the correlation):
![csa_correlated_to_weight](https://user-images.githubusercontent.com/12597538/94714927-6abd0800-0312-11eb-98ed-598600da0d8b.png)

**Comment by jcohenadad** [comment]
> I would suggest to add there also columns height_record and weight_record with 3 possible choices:

I don't find it relevant enough, and anticipate confusions with this entry. I think we can assume that people know their weight/height (the same way they know their age). Moreover, choice 3 is implied by height/weight having the value "-"

> I would also suggest to add units into the table header.

according to the BIDS convention this information goes in [participants.json](https://github.com/spine-generic/data-multi-subject/blob/master/participants.json).

**Comment by renelabounek** [comment]
Ok but in general, I still think many people tend to fill lower weight than the reality can be. Or they have not simply weighted themselves for several years and filled the last known value. Somettimes, one year old information can easily make 10kg variance.

**Comment by jcohenadad** [comment]
> Ok but in general, I still think many people tend to fill lower weight than the reality can be. Or they have not simply weighted themselves for several years and filled the last known value. Somettimes, one year old information can easily make 10kg variance.

i still don't think we should add these columns:
- i don't think a bit of noise will be detrimental to this study...
- what would we do if we had the information about _how_ the weight was acquired? get rid of the points? weight them differently in the analysis? if so, how differently? that opens a Pandora box 
- i don't think many of the 260 subjects were physically weighted and measured before the MRI scans. I'm pretty sure none of them were, so the column would always show "choice 1", which is a bit irrelevant.

**Comment by renelabounek** [comment]
I discussed it with Igor Nestrasil, he agrees with you to keep the database as simple as possible. Just to clarify your questions:

Ad point 1: Hopefully not, but you can not be fully sure.

Ad point 2: You can filter only "exactly" measured data points if you want and investigate whether estimated values blurr your results or not. Might be important e.g. for the development of CSA normalization process decreasing inter-subject COV.

Ad point 3: I agree original 260 dataset will have mostly the choice one estimated from the questionnaire. But I thought we started to build an open-access database which can be extended in the future. Future data in the database can have preciselly measured height and weight. When the database will rise at 500 subjects then this information avilable can become important and you might observe that estimated values blurr e.g. the CSA normalization method. If the acquisition method is not available in the database, then we will not  be able to investigate this effect at 500 subjects where the distribution of the acquisition method noise will be 50%:50%.

**Comment by valosekj** [comment]
We have traced weight and height for `brnoCeitec` site:

```
sub-brnoCeitec01 4.2.2019, M, 32: 187cm, 92kg
sub-brnoCeitec02 12.3.2019, F, 27: 163cm, 56kg
sub-brnoCeitec03 12.3.2019, M, 27: 186, 84kg
sub-brnoCeitec04 13.3.2019, F, 28: 170cm, 63kg
sub-brnoCeitec05 15.3.2019, F, 41: 172cm, 58kg
sub-brnoCeitec06 15.3.2019, M, 27: 186cm, 70kg
```

During searching for weight and height, we find out two things to note:

- age in [participants.tsv](https://github.com/spine-generic/data-multi-subject/blob/master/participants.tsv#L32) does not correspond for all `brnoCeitec` subjects precisely (e.g. for `sub-brnoCeitec03`, `04`, `05`, `06`). Age above is extracted directly from MR console and is related to date of scan. I do not know how age in  `participants.tsv` was gained, maybe by computation based on year of born and year of MRI scan? Then discrepancies in age +-1 can happen.
- there is typo in date of scan for `sub-brnoCeitec02` in `participants.tsv`. Correct should be as stated above - `2019-03-12`

**Comment by renelabounek** [comment]
@valosekj heigt, weight and scan date typo updated, see more details about changes here: https://github.com/spine-generic/data-multi-subject/pull/57
Changes in ages lets at @jcohenadad or other database admins.

**Comment by renelabounek** [comment]
@valosekj @jcohenadad This can be clsoed as height and weight for all subjects have already been added and merged into master in PRs #109 and #110
